src/deps_log.cc

```diff
@@ -218,9 +228,16 @@ bool DepsLog::Load(const string& path, State* state, string* err) {
       assert(size % 4 == 0);
       int* deps_data = reinterpret_cast<int*>(buf);
       int out_id = deps_data[0];
-      int mtime = deps_data[1];
-      deps_data += 2;
-      int deps_count = (size / 4) - 2;
+      TimeStamp mtime;
+      if (little_endian) {
+        mtime = (TimeStamp)((uint64_t)deps_data[2] << 32 |
+                            (uint64_t)deps_data[1]);
+      } else {
+        mtime = (TimeStamp)((uint64_t)deps_data[1] << 32 |
+                            (uint64_t)deps_data[2]);
+      }
```

Comment:
The Windows link above says not to do that because of alignment problems; I think that's a large enough platform to worry about.